lib/SauceLabs.js

```diff
@@ -268,6 +270,9 @@ function generateLink(hostname, auth, date, job) {
 }
 
 function makeRequest(options, body, callback) {
+    if(process.env.https_proxy !=='undefined' && process.env.https_proxy) {
```

Comment:
is it actually possible to not use environment variables? We don't anywhere else and we pass things into the saucelabs constructor at the beginning?